MastodonSDK/Sources/MastodonCore/Persistence/FileManager+Timeline.swift

```diff
@@ -1,22 +1,21 @@
 // Copyright © 2023 Mastodon gGmbH. All rights reserved.
 
 import Foundation
-import MastodonCore
 import MastodonSDK
 
 extension FileManager {
```

Comment:
You could also just make the extension public, then all function will implicitly inherit the access modifier.